source/__tests__/integration/create-resource.test.js

```diff
@@ -52,9 +52,9 @@ describe('Create profile resource template requirements', () => {
         resourceURI: "htt",
       }).catch(error => console.log(`promise error for filling profile form: ${error}`));
       await page.waitFor(1000)
```

Comment:
is it your intent to deal with these unconditional page.waitFor() calls in a separate PR?  Because they could be addressed here as waitForSelector(xx, {timeout: 1000})  and then do the expect on the same selector.  I could pair w you if it would be helpful.

Reply:
Hi @ndushay, I added a separate ticket to address the waitFor issues with #137

Reply:
If you look at build 680, some of the tests are not completing but marked as passing
" PASS  source/__tests__/integration/create-resource.test.js (6.751s)
  ● Console

    console.log source/__tests__/integration/create-resource.test.js:81
      promise error checkURL: Error: Protocol error (Runtime.callFunctionOn): Target closed.
    console.log source/__tests__/integration/create-resource.test.js:127
      promise error for filling profile form: Error: Protocol error (Input.dispatchKeyEvent): Target closed.
"
 "PASS  source/__tests__/integration/empty-profile-export.test.js
  ● Console

    console.log source/__tests__/integration/empty-profile-export.test.js:18
      promise error for closing alert modal: Error: Protocol error (Runtime.callFunctionOn): Target closed."

Adding time seemed to help me but it probably was just a bandaid. Prob. figuring out why the target is closing before the tests complete will help resolve this issue. Plus if a test doesn't complete shouldn't that be a fail? Maybe set fail first, then change to pass when they complete?